run_tests.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+import pytest
+
+if __name__ == "__main__":
+    os.environ['APP_SETTINGS'] = 'TestingConfig'
+    pytest.main(['--cov-report', 'term-missing', '--cov', 'frontstage_api'])
```

Comment:
Has the readme been updated to reflect this?

Reply:
yep